src/cli_plugin/install/settings.test.js

```diff
@@ -65,7 +65,7 @@ describe('parse function', function () {
   const defaultOptions = { pluginDir: fromRoot('plugins') };
   const osdPackage = { version: 1234 };
 
-  afterAll(() => {
+  beforeAll(() => {
```

Comment:
Great catch! But would it be safer to define `beforeAll` to be `linux` and `x64` and keep the `ORIGINAL_PLATFORM`, `ORIGINAL_ARCHITECTURE`, and `afterAll` the same. I specifically remember an issue with `artifact.test.js` that when I set `process` there were test failures that now caused it return the value I set when I didn't intend to.

Reply:
We would still run into the same problem in that case. If we define the platform and architecture by the local process then tests will continue to be inconsistent between machines.

Reply:
Sorry about the confusion but I believe it should still be the same?

For example: 
```
const ORIGINAL_PLATFORM = process.platform;
const ORIGINAL_ARCHITECTURE = process.arch;

...

 beforeAll(() => {
    Object.defineProperties(process, {
      platform: {
        value: 'linux',
      },
      arch: {
        value: 'x64',
      },
    });
  });
  
  
  afterAll(() => {
    Object.defineProperties(process, {
      platform: {
        value: ORIGINAL_PLATFORM,
      },
      arch: {
        value: ORIGINAL_ARCHITECTURE,
      },
    });
  });
```

Otherwise I would recommend deleting `ORIGINAL` variable since they are used to ensure that this test is manually overwriting the `process` values and resetting it back to be the original pocess platform and arch.

Reply:
Ah I see what you mean. Yeah let me try that out.

Reply:
I simplified a bit by explicitly adding the platform and arch values at the beginning of each test.